lib/matplotlib/__init__.py

```diff
@@ -1491,6 +1491,17 @@ def verify_test_dependencies():
     if not os.path.isdir(os.path.join(os.path.dirname(__file__), 'tests')):
         raise ImportError("matplotlib test data is not installed")
 
+    # The version of freetype to install locally for running the
+    # tests.  This must match the value in `setupext.py`
+    LOCAL_FREETYPE_VERSION = '2.5.2'
+
+    from matplotlib import ft2font
+    if (ft2font.__freetype_version__ != LOCAL_FREETYPE_VERSION or
+        ft2font.__freetype_build_type__ != 'local'):
+        raise ImportError(
```

Comment:
@QuLogic: You don't need a specific version of freetype installed.  You need to build matplotlib with the special "local_freetype" option, which will download and build against a particular version of freetype for you.

@jkseppan: Maybe a warning is the right thing to do -- but knownfail doesn't always communicate "the test run you just ran should give you very little confidence that anything is working" which is really the situation we would have here.  What about a warning + real failures?